node/resolvers/Mutations/Users.ts

```diff
@@ -180,6 +180,17 @@ export const addUser = async (_: any, params: any, ctx: Context) => {
   } = ctx
 
   try {
+    const costCenter = await ctx.clients.organizations.getCostCenterById(
+      params.costId
+    )
+
+    if (
+      !costCenter?.data?.getCostCenterById.name ||
+      params.orgId !== costCenter.data.getCostCenterById.organization
```

Comment:
can we simplify this to:
```suggestion
      params.orgId !== costCenter?.data?.getCostCenterById?.organization
```